src/test/scala/io/citrine/lolo/learners/RandomForestTest.scala

```diff
@@ -161,6 +163,67 @@ class RandomForestTest {
     learner.train(trainingData).getModel()
   }
 
+  def shapleyCompare(
+                      trainingData: Seq[(Vector[Any],Double)],
+                      evalLocation: Vector[Any],
+                      expected: Vector[Double]
+                    ): Unit = {
+    assert(trainingData.length >= 256, "This test is likely to fail randomly when training data are too small.")
```

Comment:
Is this still the case, given the random seeds?

Reply:
Good catch